src/TitleReceive/TitleReceiveList.js

```diff
@@ -9,78 +9,66 @@ import {
   TextArea,
   TextField,
 } from '@folio/stripes/components';
-import { LocationLookup } from '@folio/stripes/smart-components';
 import {
-  FieldSelectFinal,
+  FieldLocationFinal,
   getItemStatusLabel,
 } from '@folio/stripes-acq-components';
 
 import { PIECE_FORMAT_LABELS } from '../common/constants';
 
 const visibleColumns = ['checked', 'barcode', 'format', 'hasRequest', 'comments', 'location', 'itemStatus', 'callNumber'];
 
-export const TitleReceiveList = ({ fields, props: { locationOptions, selectLocation, toggleCheckedAll } }) => {
+export const TitleReceiveList = ({ fields, props: { selectLocation, toggleCheckedAll } }) => {
   const field = fields.name;
-  const cellFormatters = useMemo(
```

Comment:
you removed `useMemo`, I wonder if there a reason for that?

Reply:
Previously it was based on `locationOptions`

Reply:
yes, but now it's re-created every time